compiler/src/main/java/com/squareup/anvil/compiler/Utils.kt

```diff
@@ -87,6 +91,11 @@ internal fun ConstantValue<*>.toType(
 // When the Kotlin type is of the form: KClass<OurType>.
 internal fun KotlinType.argumentType(): KotlinType = arguments.first().type
 
+internal fun FqName.isAnvilModule(): Boolean {
+  val string = asString()
```

Comment:
"name" instead of "string"? I know it's a bit obvious here but if you're going to declare an intermediate variable might as well name it better 😄